packages/lit-node-client-nodejs/src/lib/lit-node-client-nodejs.ts

```diff
@@ -1971,36 +1971,22 @@ export class LitNodeClientNodeJs
   };
 
   /**
-   * Get session signatures for a set of resources
+   * This is one of the methods that can be used to get session signatures. 
    *
-   * High level, how this works:
-   * 1. Generate or retrieve session key
-   * 2. Generate or retrieve the wallet signature of the session key
+   * How this function works on a high level:
+   * 1. Generate or retrieve session keys (a public and private key pair)
```

Comment:
Link to https://v6-api-doc-lit-js-sdk.vercel.app/interfaces/types_src.SessionKeyPair.html